group.go

```diff
@@ -21,202 +21,166 @@
 package pitaya
 
 import (
-	"sync"
+	"context"
 	"sync/atomic"
 
 	"github.com/topfreegames/pitaya/constants"
+	"github.com/topfreegames/pitaya/groups"
 	"github.com/topfreegames/pitaya/logger"
-	"github.com/topfreegames/pitaya/session"
-	"github.com/topfreegames/pitaya/util"
 )
 
 const (
 	groupStatusWorking = 0
 	groupStatusClosed  = 1
 )
 
-// SessionFilter represents a filter which is used to filter sessions when Multicast,
-// the session will receive the message when the filter returns true.
-type SessionFilter func(*session.Session) bool
-
-// Group represents a session group which is used to manage a number of
-// sessions, data sent to the group will be sent to all sessions in it.
+// Group represents an agglomeration of UIDs which is used to manage
+// users. Data sent to the group will be sent to all users in it.
```

Comment:
This description is a little weird, `agglomeration of UIDs`? What does manage users cover?